docs/personalization/user_guide/recommendation_models.md

```diff
@@ -128,11 +128,20 @@ After the submodel is configured and built (happens automatically during the n
 
 #### Numeric Attributes
 
-Very similar configuration can be done by using a numeric attribute. 
+Very similar configuration can be done by using a numeric attribute.
```

Comment:
With the sentence below this one is redundant and unnecessary.

Reply:
Good point!